src/components/input-message/input-message.tsx

```diff
@@ -48,8 +51,13 @@ export class InputMessage {
   //--------------------------------------------------------------------------
 
   connectedCallback(): void {
-    this.scale = getElementProp(this.el, "scale", this.scale);
-    this.requestedIcon = setRequestedIcon(StatusIconDefaults, this.icon, this.status);
+    const content = this.el.textContent;
+    if (content === "" && !this.icon) {
```

Comment:
I think we will need to setup a mutationObserver to watch for `textConttent` changes. Otherwise, if a user adds or removes `textContent` after the component `connectedCallback` has been called then this if statement won't change. 

We need conventions on this as well. eek :(

I'll make a note for these conventions to be added.